### PR TITLE
Top languages

### PR DESCRIPTION
**Pull request by xeBuz**
Updated the list of languages, to get the most relevants on top. The list was taken from an older version from this repository

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -61,16 +61,27 @@ <h1 class="text-center text-5xl font-bold mb-8">Find issues to help with during
                                 v-model="selectedLanguage"
                                 class="block appearance-none w-full bg-white px-4 py-2 pr-8 rounded leading-tight focus:outline-none focus:shadow-outline"
                             >
+
+                              <option selected value="any">Any language</option>
```
Comment (damcclean) [inline index.html:65]: Would you mind reverting the indentation on this element please?

File: src/js/app.js [hunk 2]
```diff
@@ -104,7 +105,7 @@ new Vue({
             return issues.sort((a, b) => {
                 if (this.selectedSort==='noReplies')
```
Comment (damcclean) [inline src/js/app.js:106]: I know this isn't your fault but would you mind adding a `{` to the end of this line and do the same sort of thing for the else statement, so it's like this

```js
if (whatever === 'something') {
    console.log('whatever is something')
} else {
    console.log('whatever is not something')
}
```

**Review by damcclean — CHANGES_REQUESTED** [review]
Thanks for your pull request! I've left a few comments, if you wouldn't mind having a look over them, that would be great.

One tiny thing I noticed is that when you load the page initially, it looks a little differently than it should.

**This branch**
image

**The current version (master)**
image
Also note how the load more button doesn't display on the current version.

Hopefully that's something nice and easy you could fix. Thanks again for the PR!

**Comment by xeBuz** [comment]
> 
> One tiny thing I noticed is that when you load the page initially, it looks a little differently than it should.
> 
> Also note how the load more button doesn't display on the current version.
> 
> Hopefully that's something nice and easy you could fix. Thanks again for the PR!

This is weird, I can't get the same behaviour using the deploy preview version: https://deploy-preview-211--hacktoberfest-finder.netlify.app/

**Comment by damcclean** [comment]
> > One tiny thing I noticed is that when you load the page initially, it looks a little differently than it should.
> > Also note how the load more button doesn't display on the current version.
> > Hopefully that's something nice and easy you could fix. Thanks again for the PR!
> 
> This is weird, I can't get the same behaviour using the deploy preview version: https://deploy-preview-211--hacktoberfest-finder.netlify.app/

That's strange, that's where I'm seeing it. I'm using Chrome if that makes any difference.

**Comment by damcclean** [comment]
The other weird thing is that it doesn't look like you've touched anything that would affect it...

File: index.html [hunk 3]
```diff
@@ -61,16 +61,30 @@ <h1 class="text-center text-5xl font-bold mb-8">Find issues to help with during
                                 v-model="selectedLanguage"
                                 class="block appearance-none w-full bg-white px-4 py-2 pr-8 rounded leading-tight focus:outline-none focus:shadow-outline"
                             >
+
```
Comment (damcclean) [inline index.html:64]: Sorry to be really pernickety, could you remove the whitespace either side of the first and last elements in this `select` element? 

(sorry)

**Review by damcclean — CHANGES_REQUESTED** [review]

**Comment by damcclean** [comment]
Thanks for the change! I'll try and remember to pull down later so I can see if the weird lazy loading issues happen locally.

**Comment by xeBuz** [comment]
@damcclean is there anything I can do here?

**Comment by damcclean** [comment]
> @damcclean is there anything I can do here?

Ah, sorry! Must have forgotten about this PR. I don't think there's anything you need to do 😂

**Comment by damcclean** [comment]
> @damcclean is there anything I can do here?

Ahhh, sorry, must have forgotten about this pull request! I've just cloned this pull request down locally and recompiled the assets and I'm still seeing the same issue with the lazy loading stuff. 

Would you mind trying to reproduce it again and have a look at fixing it? If it's affecting the load more button too, maybe its a JavaScript thing rather than something to do with the HTML/CSS.

**Comment by damcclean** [comment]
I'm going to merge this pull request now. We can fix that lazy loading bug another time. 

Thanks for your hard work and sorry for the long wait to get this merged.

**Pull request merged**